packages/cli/src/util/logger.js

```diff
@@ -4,34 +4,48 @@
 import chalk from 'chalk';
 
 const SEPARATOR = ', ';
+const TITLE = '[rn-cli]';
```

Comment:
I'm still not sure about this. E.g. Yarn doesn't output stuff like that, but instead it reprints the command like this:

![image](https://user-images.githubusercontent.com/5106466/52375917-21df7e00-2a61-11e9-8b16-1c83a94c2c80.png)

Maybe we should go this way? I find it nice

Reply:
Good enough?
<img width="389" alt="zrzut ekranu 2019-02-6 o 22 54 03" src="https://user-images.githubusercontent.com/9092510/52376365-425c0800-2a62-11e9-9ad3-88b8cbb3c153.png">

Reply:
The simpler the better it seems :P

Reply:
Ship it